nox/virtualenv.py

```diff
@@ -62,6 +62,24 @@ def run(self, args, in_venv=True):
             path=self.bin if in_venv else None).run()
 
 
+def locate_via_py(version):
+    """Find the Python executable using the Windows launcher.
+
+    This is based on :pep:397 which details that executing
+    ``py.exe -{version}`` should execute python with the requested
+    version. We then make the python process print out its full
+    executable path which we use as the location for the version-
+    specific Python interpreter.
+    """
+    script = "import sys; print(sys.executable)"
+    py_exe = py.path.local.sysfind('py')
+    if py_exe:
```

Comment:
@jonparrott Should this be a [`None`-check][1]?

[1]: http://iruntheinternet.com/lulzdump/images/nunchucks-nuns-chuck-norris-nun-12593488599.jpg

Reply:
Possibly. If you want you can send a follow-up PR.

Reply:
Done #59